spec/requests/users_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  before :each do
+    user = User.create(name: 'Akite Donald', posts_counter: 0)
+    post = user.posts.create(title: 'Post 1', comments_counter: 0, likes_counter: 0)
+    post.author_id = user.id
+    post.save!
+  end
+  describe 'GET /users' do
+    it 'works! test root' do
+      get '/users'
+      expect(response).to have_http_status(200)
+    end
+    it 'renders the correct template' do
+      get '/users'
+      expect(response).to render_template('index')
+    end
+    it 'response body includes the correct content' do
+      get '/users'
+      expect(response.body).to match(/<h1>this is the home page <h1>/)
+    end
+  end
+  describe 'GET /users/:id' do
+    let(:user) { User.create(name: 'Akite Donald', posts_counter: 0) }
+    it 'has 200 status code' do
+      get '/users', params: { id: user.id }
+      expect(response).to have_http_status(200)
+    end
+    it 'renders the right template' do
+      get '/users/1'
+      expect(response).to render_template('show')
+    end
+  end
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, please make sure to include a test for the `response.body` containing the correct placeholder text, just as you did in your `/users` tests! 

According to the requirements:
![image](https://github.com/quavo19/social-blog/assets/77069011/64f24e44-e2e8-46c3-8884-10f5c23c4ea7)

Please consider including this! 🤓 🔧